control-userlayer/src/main/java/org/oskari/control/userlayer/UserLayerHandlerHelper.java

```diff
@@ -23,5 +29,48 @@ public static UserLayer getUserLayer(UserLayerDbService service, ActionParameter
         }
         return userLayer;
     }
-
+    public static JSONObject createErrorJSON (String error, String key, Set<String> set) {
```

Comment:
All these JSON helper functions here seem a bit off. Maybe consider an `UserLayerException` class that extends ServiceException and then move all the functionality in these functions to methods inside that exception class.